### PR TITLE
Fix Bug 1149620 - Firefox /all/ download page now has Windows stubs instead of full installers

### PR DESCRIPTION
**Pull request by kyoshino**

### CODE REVIEW

File: bedrock/firefox/firefox_details.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ def _get_filtered_builds(self, builds, channel, version=None, query=None):
                 build_info['platforms'][platform] = {
                     'download_url': self.get_download_url(channel, version,
                                                           platform, locale,
-                                                          True),
+                                                          True, True),
```
Comment (pmclanahan) [inline bedrock/firefox/firefox_details.py]: Won't this mean that nothing gets stub?

**Comment by kyoshino** [comment]
@pmclanahan r? 

I'll send a separate PR for mcom-tests.

File: bedrock/firefox/firefox_details.py (hunk 1 above)
Comment (kyoshino) [inline bedrock/firefox/firefox_details.py]: Yes, this is in `_get_filtered_builds` and the function is only for the /all/ pages. The download buttons are not affected.

Comment (pmclanahan) [inline bedrock/firefox/firefox_details.py]: Ah yes! Of course your're right. Thanks.

**Comment by pmclanahan** [comment]
If you have time I'd feel a lot better if there were a regression test with this. Is that what you plan to add to mcom-tests?

**Comment by kyoshino** [comment]
Yes, I'll add something to mcom-tests.

**Pull request merged**